boards/arm/nrf52_adafruit_feather/Kconfig

```diff
@@ -0,0 +1,13 @@
+# Kconfig - nRF52 Adafruit Feather board configuration
+#
+# Copyright (c) 2018 LEDCity AG
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF52_ADAFRUIT_FEATHER
+
+config  GPIO_AS_PINRESET
```

Comment:
Remove this kconfig entry. Recently #7449 generalized this for all nRF52 SoC - so it became the SoC option.

Reply:
@nvlsianpu removed and squashed